colonialsite/colonialsite/settings.py

```diff
@@ -15,59 +15,109 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Path to file uploads (upload images)
-MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
-
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '-j_7bebp&w9&()_nn+*dmg93%5qxy#$8tz)()2e-d&=ne#767p'
+# --- IMPORTANT SETTINGS ---
 
-# SECURITY WARNING: don't run with debug turned on in production!
+# SECRET_KEY
 DEBUG = True
 
-ALLOWED_HOSTS = []
+# EMAIL_HOST_USER
+# EMAIL_HOST_PASSWORD
 
-# --- NEW SETTINGS ---
-
-SOPHOMORE_YEAR = '2019'
-JUNIOR_YEAR = '2018'
-SENIOR_YEAR = '2017'
+# TIGERBOOK_LOGIN
+# TIGERBOOK_PW
 
 # Used in coloauth.middleware
-# Default Login is automatically added
+# Default Login is automatically added 
 # in RE format
 LOGIN_EXEMPT_URLS = [r'^accounts/logout($|.*)', r'^staff/login($|.*)', r'^staff/logout($|.*)',]
 
 STAFF_LOGIN_URL = '/staff/login/'
 STAFF_LOGIN_REDIRECT = '/announcements'
 
-# --- END NEW SETTINGS ---
 
-# Application definition
+# --- APPLICATIONS AND MIGRATIONS ---
 
 INSTALLED_APPS = [
     'announcements.apps.AnnouncementsConfig',
-    'reservations.apps.ReservationsConfig',
     'coloauth.apps.ColoauthConfig',
     'dashboard.apps.DashboardConfig',
     'events.apps.EventsConfig',
-    'menus.apps.MenusConfig',
     'members.apps.MembersConfig',
+    'menus.apps.MenusConfig',
+    'reservations.apps.ReservationsConfig',
     'webpack_loader',
     'django.contrib.admin',
     'django.contrib.auth',
-    'rest_framework',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'rest_framework',
     'cas',
     'django_filters',
 ]
 
+
+# --- URLS ---
+
+LOGIN_URL = '/accounts/login/'
+CAS_SERVER_URL = 'https://fed.princeton.edu/cas/'
+ROOT_URLCONF = 'colonialsite.urls'
+
+
+# --- EMAIL AND LOGGING ---
+
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
Should this be in the dev settings? Are you going to start sending failure logs from dev environments?

Reply:
Logging is removed from local settings but it's fair that these settings are unnecessary; I'll move them to production_settings only